Source/Core/DolphinQt/GCMemcardCreateNewDialog.cpp

```diff
@@ -0,0 +1,87 @@
+// Copyright 2019 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DolphinQt/GCMemcardCreateNewDialog.h"
+
+#include <QComboBox>
+#include <QDialogButtonBox>
+#include <QFileDialog>
+#include <QGridLayout>
+#include <QLabel>
+#include <QPushButton>
+#include <QRadioButton>
+
+#include "Common/FileUtil.h"
+#include "Common/MsgHandler.h"
+
+#include "Core/HW/GCMemcard/GCMemcard.h"
+
+GCMemcardCreateNewDialog::GCMemcardCreateNewDialog(QWidget* parent) : QDialog(parent)
+{
+  m_combobox_size = new QComboBox();
+  m_combobox_size->addItem(tr("4 Mbit (59 blocks)"), 4);
+  m_combobox_size->addItem(tr("8 Mbit (123 blocks)"), 8);
+  m_combobox_size->addItem(tr("16 Mbit (251 blocks)"), 16);
+  m_combobox_size->addItem(tr("32 Mbit (507 blocks)"), 32);
+  m_combobox_size->addItem(tr("64 Mbit (1019 blocks)"), 64);
+  m_combobox_size->addItem(tr("128 Mbit (2043 blocks)"), 128);
+  m_combobox_size->setCurrentIndex(5);
+
+  m_radio_western = new QRadioButton(tr("Western (Windows-1252)"));
+  m_radio_shiftjis = new QRadioButton(tr("Japanese (Shift-JIS)"));
+  m_radio_western->setChecked(true);
+
+  auto* card_size_label = new QLabel(tr("Card Size"));
+  auto* card_encoding_label = new QLabel(tr("Encoding"));
+  auto* button_box = new QDialogButtonBox(this);
+  QPushButton* create_button = button_box->addButton(tr("Create..."), QDialogButtonBox::AcceptRole);
```

Comment:
By telling QDialogButtonBox to give you an OK button, this can be replaced with:

```suggestion
  button_box->button(QDialogButtonBox::Ok)->setText(tr("Create..."));
```

and the following line can be removed too.